plugins/autopeering/parameters.go

```diff
@@ -14,7 +14,10 @@ var Parameters = struct {
 	Ro float64 `default:"2." usage:"Ro parameter"`
```

Comment:
```suggestion
	Ro float64 `default:"2" usage:"Ro parameter"`
```
`2.` is strange let's either go for `2` or `2.0`

Reply:
interestingly in older versions of python   `2` would define an integer and `2.` would define a float. Thus in Python i would define float variables as `someVariable = 2.`